### PR TITLE
Add Standalone server

### PR DESCRIPTION
**Pull request by catch-point**

### CODE REVIEW

**Review by dbooth-boston — APPROVED** [review]
LGTM

File: src/noflo-ui-server.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    return promiseResult(server.listen, server, program.port).then(function(port){
+        console.log('NoFlo UI is available at ' + program.url);
+        console.log('Using ' + baseDir + ' for noflo-ui');
+    });
+}).then(function(){
+    return promiseResult(process.on, process, 'SIGINT');
+}).then(function(){
+    process.exit(0);
+}).catch(function(error){
+    console.log(error);
```
Comment (glenna-m) [inline src/noflo-ui-server.js]: shouldn't this be console.error(error)?

**Comment by glenna-m** [comment]
Two requests: 
1. On the catch block, can you change console.log(error) to console.error(error)?
2. Can you add a section documenting how this repository should be set up for development, since it is a bit different from what we've documented here?

**Comment by glenna-m** [comment]
LGTM

**Pull request merged**